metadata-load/src/test/resources/assembly/assemblyDocumentAPI

```diff
@@ -0,0 +1,266 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Can you rename the xmls and text files in CamelCase and with extension eg: AssemblyDocumentAPI.xml

Reply:
Fixed in https://github.com/EBIvariation/metadata-ws/pull/50/commits/1847b04057735e88ea0a3fe099dc559c01405dd8.